### DISCUSSION TITLE
Hello there!

### DISCUSSION THREAD

**Discussion by jS5t3r** [OpenMined/courses#39] [Introduce Yourself]
Hi,

I am Peter from Austria,  I am a ML engineer. I  really excited to learn and grow in this community.

**Comment by yemikifouly** [comment]
Hi Peter! 🙂 Welcome to the discussion board! And thank you for being here. Really! It must have been very late in Austalia when we started the welcoming event. 🙂

**Reply by jS5t3r** [reply]
Thank you :)

**Comment by iamtrask** [comment]
Hi Peter! Welcome!

**Reply by jS5t3r** [reply]
Thank you :)
